bin/asset-size-tracking/generate-diff.js

```diff
@@ -23,12 +23,23 @@ const builtAsset = getBuiltDist(NEW_VENDOR_FILE);
 const new_library = parseModules(builtAsset);
 
 function getDiff(oldLibrary, newLibrary) {
+  const compressionDelta = newLibrary.compressedSize - oldLibrary.compressedSize;
+
+  function getCompressionDelta(item) {
```

Comment:
Could use a better name. getRelativeDeltaForItem?

getCompressionDelta sounds like it just returns previous line.